data/monster/monsters/retching_horror.xml

```diff
@@ -44,10 +44,10 @@
 		<element energyPercent="10" />
 		<element firePercent="85" />
 		<element icePercent="15" />
-		<element deathPercent="20" />
+		<element deathPercent="22" />
```

Comment:
accordingly to wiki, 20 is the correct value
https://tibia.fandom.com/wiki/Retching_Horror